tests/test_observe.py

```diff
@@ -102,6 +102,39 @@ def __call__(self, change):
     assert "str or callable" in excinfo.exconly()
 
 
+@pytest.mark.parametrize(
```

Comment:
We typically need an IntFlag here, since otherwise it is a mess to read to be honest.